module/documents/chat-message.mjs

```diff
@@ -4,6 +4,12 @@ import simplifyRollFormula from "../dice/simplify-roll-formula.mjs";
 
 export default class ChatMessage5e extends ChatMessage {
 
+  /**
+   * HTML tag names for chat trays that can open and close.
+   * @type {string[]}
+   */
+  static TRAY_TYPES = ["damage-application", "enchantment-application"];
```

Comment:
apologies if I am confused, but should this include effect-application?
```diff
- static TRAY_TYPES = ["damage-application", "enchantment-application"];
+ static TRAY_TYPES = ["damage-application", "effect-application"];
```
In my testing it seems the effects tray no longer responds to the `"autoCollapseChatTrays"` setting without the effect application element here, and currently the enchantment tray cannot be collapsed (at least from what I've seen in testing)